connect/runtime/src/test/java/org/apache/kafka/connect/storage/KafkaStatusBackingStoreFormatTest.java

```diff
@@ -216,7 +216,7 @@ public void putTopicStateRetriableFailure() {
         }).when(kafkaBasedLog).send(eq(key), valueCaptor.capture(), any(Callback.class));
 
         store.put(topicStatus);
-        verify(kafkaBasedLog, times(2)).send(any(), any(), any());
+        verify(kafkaBasedLog, timeout(300).times(2)).send(any(), any(), any());
```

Comment:
I may be a bit pessimistic but I'd bump this up to 1 second.

Reply:
thanks @viktorsomogyi . I created a separate PR so that the flaky test can be fixed before the code freeze(27th April). Here is the PR: https://github.com/apache/kafka/pull/13634